README.md

```diff
@@ -89,6 +90,7 @@ This action is primarily designed for use with GitHub.com's Actions workflows an
 
 | Release | GHES Compatibility |
 |:---|:---|
+| [`v4`](https://github.com/actions/deploy-pages/releases/tag/v4) | :information_source: TBD |
```

Comment:
I'd maybe go with a strong indication that it is outright NOT compatible yet. "TBD" sounds a bit like "It might be, but we need to check which version".

Something like:

```suggestion
| [`v4`](https://github.com/actions/deploy-pages/releases/tag/v4) | :warning: Incompatible. Future compatibility TBD. 🔮 |
```

Feel free to edit. 👌🏻